src/test/scala/mesosphere/marathon/integration/AppDeployIntegrationTest.scala

```diff
@@ -30,6 +30,52 @@ class AppDeployIntegrationTest
   //clean up state before running the test case
   before(cleanUp())
 
+  // FIXME (gkleiman): Docker tests only work when the integration tests are run on Linux using bin/run-tests.sh
+  if (sys.env.contains("BUILD_ID")) {
```

Comment:
this only tests if it was run via ./bin/run-tests.sh, right? If you did that on Mac OS, you would get a weird error. We do recommend running this script: https://mesosphere.github.io/marathon/docs/contributing.html#submitting-code-changes-to-marathon